website/project/model.py

```diff
@@ -4495,6 +4495,9 @@ def create_from_node(cls, node, user, schema, data=None):
         return draft
 
     def update_metadata(self, metadata):
+        if self.is_approved:
+            return {}
```

Comment:
This method typically returns a `list`, so this should be `[]`. I can make this change when I merge.